### ISSUE TITLE
Issues of training

### ISSUE THREAD

**Issue by bo-miao** [hkchengrex/STCN#108]
Hi,

May I know the training hyperparameters of 0 + 3? 

I try to first use the hyperparameters [static stage0] to train the model on static images, and then reload the model that trained on static images and further train the model on VOS datasets [VOS stage 3]. But I did not find improvement compared with only training with [VOS stage 3]. Did you change some hyperparameters for the 2-stage training, e.g. reduce lr in the second stage?

Thank you.

''' [static stage0]
self.args['lr'] = none_or_default(self.args['lr'], 1e-5)
self.args['batch_size'] = none_or_default(self.args['batch_size'], 8)
self.args['iterations'] = none_or_default(self.args['iterations'], 300000)
self.args['steps'] = none_or_default(self.args['steps'], [150000])
loss_p: step 2w to 7w
'''

'''  [VOS stage 3]
self.args['lr'] = none_or_default(self.args['lr'], 1e-5)
self.args['batch_size'] = none_or_default(self.args['batch_size'], 4)
self.args['iterations'] = none_or_default(self.args['iterations'], 150000)
self.args['steps'] = none_or_default(self.args['steps'], [125000])
loss_p: step 2w to 7w
'''

**Comment by hkchengrex** [comment]
The hyperparameters listed are the ones that I used... What performance are you getting? In Table 7 of the paper, we show that main training only is significantly worse than static+main. And that "main training" result is obtained by training only with stage 3.

**Comment by bo-miao** [comment]
When I use main training only I could get 82.x J&F which is the same as paper (Table 7 Main only), but I tried once to use static + main the performance is still 82.x (first use [static stage0] to train, and then re-load the trained model and use [VOS stage 3] to train.

I will try another time with this config. Thank you.

**Comment by hkchengrex** [comment]
Not sure what other config did you use but with pre-training you should clearly see a much better performance in the first few hundred iterations. If it doesn't then something is wrong.

**Comment by bo-miao** [comment]
Thank you for your reply.

I noticed that I also reload the "optimizer" at the same time instead of only reloading parameters, and have changed the random seed during training. Maybe that is one of the reasons.

**Comment by bo-miao** [comment]
Just an update, I could get similar performance as in paper via stage0+3 training.
